geps/gep-1897/index.md

```diff
@@ -22,18 +22,15 @@ the service or backend owner wants to validate the clients connecting to it, two
 1. The solution must satisfy the following use case: the backend pod has its own
 certificate and the gateway implementation client needs to know how to connect to the
 backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
-2. In terms of the Gateway API personas, only the application developer persona applies in this
-solution. The application developer should control the gateway to backend TLS settings,
-not the cluster operator, as requiring a cluster operator to manage certificate renewals
-and revocations would be extremely cumbersome.
-3. The solution should consider client certificate settings used in the TLS handshake **from
-Gateway to backend**, such as server name indication, trusted certificates,
-and CA certificates.
+2. Both the application developer persona, and gateway operator persona will have control over TLS settings.
```

Comment:
> and gateway operator persona

IIUC this moved to a separate GEP. please correct me if misunderstood.  so should we move this to long term goal?

Reply:
I will mention this is a future goal thanks!

Reply:
```suggestion
2. In this GEP, only the application developer persona will have control over TLS settings. This does not preclude adding other personas in future GEPs.
```

Or something like that maybe?